claws/openclaw/sageox-summary/scripts/install-ox-git.sh

```
@@ -0,0 +1,137 @@
+  *)
+    echo "error: clone path must be absolute and under \$HOME: $CLONE_PATH" >&2
+    exit 2
+    ;;
+esac
+case "$CLONE_PATH" in
+  *..*)
+    echo "error: clone path must not contain '..': $CLONE_PATH" >&2
+    exit 2
+    ;;
+esac
+for ch in ';' '$' '`' '|' '&' '<' '>' '(' ')' '{' '}' '*' '?' '[' ']' '!' '\'; do
```

Shellcheck (0.11.0)

[info] Want to escape a single quote? echo 'This is how it'\''s done'.

(SC1003)